src/features/dailyNotesPopup.tsx

```diff
@@ -656,13 +627,60 @@ export const component = {
   },
 };
 
+let unloads = new Set<() => void>();
 export let enabled = false;
-export const toggleFeature = (flag: boolean) => {
+export const toggleFeature = (flag: boolean, extensionAPI: OnloadArgs["extensionAPI"]) => {
   enabled = flag;
   if (flag) {
+    unloads.add(
```

Comment:
RE: 
> My biggest question is why did we remove all of the hotkeys to the commands?
> 
> The goal of the task was to make all of the commands have configurable hotkeys, while still retaining the default value we had so that there's no interruption in functionality

The first reason was that the docs recommend against it, so I was waiting for your thoughts on that.

> Also, the [docs](https://roamresearch.com/#/app/developer-documentation/page/HlEqgAw3H) seem to strongly recommend against setting a default shortcut. Do we still want to go ahead with that?

The second was that I was considering the UX from a new user.  As a new user, lets say I see some features in Workbench that I want to implement in my graph.  I install Workbench that comes preloaded with multiple modules and hotkey shortcuts.  I have to either knowingly turn off the modules I don't want and unset the preconfigured hotkeys or, what seems more likely, unknowingly have some undesired effects down the line.

That being said, I understand causing causing a current user interruptions is not good either.

Neither situation (removing defaults vs setting them), in my opinion, is ideal.  But with the the latter it is a singular occurrence that we can preemptively mitigate by messaging on Twitter / Slack / Email list.  The former is ongoing.

Reply:
So I think it's valid to disregard going forward. But one lesson I've learned is that no amount of messaging will resolve an interruption of behavior.

Particularly ones like `hot_keys`/`jumpnav` which all it does is add more hot keys for the user.

 We are also not precluded from removing it later on if it turns out that too many defaults is not desirable